libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -601,14 +601,14 @@ bool RawSiteToSiteClient::transmitPayload(const std::shared_ptr<core::ProcessCon
       deleteTransaction(transactionID);
     context->yield();
     tearDown();
-    logger_->log_debug("Caught Exception %s", exception.what());
+    logger_->log_debug("Caught Exceptionduring RawSiteToSiteClient::transferBytes, type: %s, what: %s", typeid(exception).name(), exception.what());
```

Comment:
this seems to be the `transmitPayload` method

Reply:
fix this and others in [8ffe9c6](https://github.com/apache/nifi-minifi-cpp/pull/1436/commits/8ffe9c6d734750b583de717c74362b3fdb865f1a)